pubsub/aws/snssqs/snssqs.go

```diff
@@ -41,23 +40,23 @@ type snsSqsMetadata struct {
 	sqsQueueName string
 
 	// aws endpoint for the component to use.
-	awsEndpoint string
-	// aws account ID to use for SNS/SQS. Required
-	awsAccountID string
-	// aws secret corresponding to the account ID. Required
-	awsSecret string
-	// aws token to use. Required
-	awsToken string
-	// aws region in which SNS/SQS should create resources. Required
-	awsRegion string
+	Endpoint string
```

Comment:
i think changing the name here makes this a breaking-change. 
cc @yaron2 ?
the names make it consistent with other aws components but the consistency is lost within the same struct. 
Should we consider capitalizing the other properties also?

Reply:
If a field needs to be exported, it should be capitalized. that's the Go rule..

Reply:
@mukundansundar yes it's a breaking change. However, I've implemented an aliasing thingy so Component manifests using the old attribute names should still work.